mods/lord/Player/clans/src/clans.lua

```diff
@@ -223,7 +229,7 @@ return {
 
 		register_callbacks()
 
-		register_nametag_operations()
+		register_join_or_leave_operations()
```

Comment:
тогда сюда перекинь 198-ю строку: `require("clans.players.nametag")`
```suggestion
        require("clans.players.nametag")

		register_join_or_leave_operations()
```
а там грохни